ui/ui.go

```diff
@@ -279,6 +291,9 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 			return m, cmd
 
+            // I do not think this state machine works quite well enough with custom keybinds
+            // as it kinda breaks down if you use two different keys across the views.
+            // Is it worth refactoring this to first check the "view" before the key?
```

Comment:
@dlvhdr just a question here about the keybind checks. I noticed we are checking for both PR and issue keys at once which sort of breaks if you bind to two separate keys. Wanted to clarify before I made any major changes.

Reply:
Yeah you're right, we would have to first check the current view

Reply:
Do you want me to make that change now or leave it for you?

Reply:
Making the change would be very appreciated 🔥